apisix/plugins/openid-connect.lua

```diff
@@ -471,7 +473,7 @@ local function required_scopes_present(required_scopes, http_scopes)
 end
 
 function _M.rewrite(plugin_conf, ctx)
-    local conf = core.table.clone(plugin_conf)
+    local conf = fetch_secrets(plugin_conf)
```

Comment:
```suggestion
    local conf = core.table.clone(plugin_conf)
    local conf = fetch_secrets(plugin_conf)
```

Reply:
@darkSheep404, why did you resolve this conversation by yourself? You must not remove `core.table.clone`. As it is a bugfix: https://github.com/apache/apisix/issues/1956